platform/jvm/Logging.kt

```diff
@@ -0,0 +1,16 @@
+package com.vaticle.bazel.distribution.platform.jvm
+
+import com.vaticle.bazel.distribution.platform.jvm.Logging.LogLevel.DEBUG
+
+object Logging {
+    class Logger(private val logLevel: LogLevel) {
+        fun debug(message: () -> String) {
+            if (logLevel == DEBUG) println(message())
+        }
+    }
+
+    enum class LogLevel {
```

Comment:
To be honest I can't see much reason to support a broader range of log levels than DEBUG and ERROR for a build system.

I suppose it would make sense if the process was really complex - but ours isn't. It's one function.